android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -2008,28 +2030,37 @@ public static void removeKeyboardEventListener(OnKeyboardEventListener listener)
     KMKeyboard.removeOnKeyboardEventListener(listener);
   }
 
-  public static int getOrientation(Context context) {
+  /* package-private */
+  static void setOrientation(int orientationFlag) {
+    orientation = orientationFlag;
+  }
+
+  public static int detectOrientation(Context context) {
```

Comment:
We probably need to keep the API call getOrientation() which can just call detectOrientation()

Documentation wip on keymanapp/help.keyman.com#1270